fabfile.py

```diff
@@ -0,0 +1,68 @@
+import os
+import yaml
+
+from fabric.api import env, run
+
+
+# Load in server config, here's a quick example:
+#   role_name:
+#     hosts:
+#       - ubuntu@12.34.56.78
+#     broker_url: pyamqp://user:pass@host:port/vhost-gpu
+#     is_gpu: true
+#     docker_image: codalab/competitions-v2-compute-worker:nvidia
+#
+# You select the role to run with like s:
+#   $ fab -R role_name <command>
+env.roledefs = yaml.load(open('server_config.yaml').read())
```

Comment:
do we need to specify a yaml loader here? (assuming we don't since it would have yelled at you, but just confirming.)

Reply:
I didn't see any warnings (they could be squelched?) and it's user defined so I _think_ we should be ok, that's a good spot though